### ISSUE TITLE
Only variables should be passed by reference

### ISSUE THREAD

**Issue by maxyc** [phpnode/YiiRedis#23]
i have promlem

```
PHP Error[2048]: Only variables should be passed by reference
    in file /var/www/***/app/lib/vendor/codemix/yiiredis/ARedisIterableEntity.php at line 29
```

php -v

```
PHP 5.5.3-1ubuntu2.6 (cli) (built: Jul  7 2014 16:54:53) 
Copyright (c) 1997-2013 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2013 Zend Technologies
    with XCache v3.1.0-dev, Copyright (c) 2005-2013, by mOo
    with Zend OPcache v7.0.3-dev, Copyright (c) 1999-2013, by Zend Technologies
    with Xdebug v2.2.3, Copyright (c) 2002-2013, by Derick Rethans
    with XCache Optimizer v3.1.0-dev, Copyright (c) 2005-2013, by mOo
    with XCache Cacher v3.1.0-dev, Copyright (c) 2005-2013, by mOo
    with XCache Coverager v3.1.0-dev, Copyright (c) 2005-2013, by mOo
```

use:

```
$user = new ARedisHash('user.'.$userResourceId);
$out[]=$user->toArray();// ->getData()
```

**Comment by maxyc** [comment]
i change 
codemix/yiiredis/ARedisIterableEntity.php at line 29

```
    public function getIterator()
    {
        $data = $this->getData();
        return new CListIterator($data);
    }
```

**Comment by phpnode** [comment]
closed via #24, thanks
